src/worker_bundle.rs

```diff
@@ -0,0 +1,107 @@
+use flate2::write::ZlibEncoder;
+use flate2::Compression;
+use number_prefix::{NumberPrefix, Prefixed, Standalone};
+use reqwest::multipart;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::{Path, PathBuf};
+
+use crate::settings::binding::Binding;
+use crate::settings::metadata;
+use crate::terminal::message;
+
+const WORKER_BUNDLE_OUT: &str = "./worker";
+
+#[derive(Debug)]
+pub struct WorkerBundle {
+    // out: String,
+    pub script_path: PathBuf,
+    pub wasm_path: Option<PathBuf>,
```

Comment:
It seems like the wasm stuff should get owned by a binding rather than being top-level attributes of a WorkerBundle.

Reply:
Your approach make sense to me now. I was hesitant to modify the binding struct because it doesn't strictly match with the online bindings anymore, but I guess it's fine.